resources/metadata/gctsCreateRepository.yaml

```diff
@@ -0,0 +1,100 @@
+metadata:
+  name: gctsCreateRepository
+  description: Creates a Git repository on an ABAP system
+  longDescription: |
+    Creates a local Git repository on an ABAP system if it does not already exist.
+
+spec:
+  inputs:
+    secrets:
+      - name: credentialsId
+        description: Jenkins credentials ID containing username and password for authentication to the ABAP system on which you want to create the repository
+        type: jenkins
+    params:
+      - name: username
+        type: string
+        description: Username to authenticate to the ABAP system
+        scope:
+        - PARAMETERS
+        - STAGES
+        - STEPS
+        mandatory: true
+        aliases:
+          - name: username
+      - name: password
+        type: string
+        description: Password to authenticate to the ABAP system
+        scope:
+        - PARAMETERS
+        - STAGES
+        - STEPS
+        mandatory: true
+        aliases:
+          - name: password
+      - name: repositoryName
+        type: string
+        description: Specifies the name (ID) of the local repository
+        scope:
+        - PARAMETERS
+        - STAGES
+        - STEPS
+        mandatory: true
+        aliases:
+          - name: repositoryName
+      - name: host
+        type: string
+        description: Specifies the host address of the ABAP system including the port
+        scope:
+        - PARAMETERS
+        - STAGES
+        - STEPS
+        mandatory: true
+        aliases:
+          - name: host
+      - name: client
+        type: string
+        description: Specifies the client of the ABAP system to be adressed
+        scope:
+        - PARAMETERS
+        - STAGES
+        - STEPS
+        mandatory: true
+        aliases:
+          - name: client
+      - name: githubURL
+        type: string
+        description: URL of the corresponding remote repository
+        scope:
+        - PARAMETERS
+        - STAGES
+        - STEPS
+        aliases:
+          - name: githubURL
+      - name: role
+        type: string
+        description: Role of the local repository. Choose between 'TARGET' and 'SOURCE'. Local repositories with a TARGET role will NOT be able to be the source of code changes.
+        scope:
+        - PARAMETERS
+        - STAGES
+        - STEPS
+        aliases:
+          - name: role
+      - name: vSID
+        type: string
+        description: Virtual SID of the local repository. The vSID corresponds to the transport route that delivers content to the remote Git repository.
+        scope:
+        - PARAMETERS
+        - STAGES
+        - STEPS
+        aliases:
+          - name: vSID
+      - name: type
+        type: string
+        description: Type of the used source code management tool.
+        scope:
+        - PARAMETERS
+        - STAGES
+        - STEPS
+        default: GIT
+        aliases:
+          - name: type
```

Comment:
What is the reason of defining an *alias* with the same name?

Reply:
No reason.. copied from other examples tbh

Reply:
yes